src/Nordea/Components/MultiSelectDropdown.elm

```diff
@@ -60,99 +72,99 @@ init { onFocus, newOutsideClickListener } =
-                        display none
-                    , flexDirection column
-                    , backgroundColor Colors.white
+        hasTextInput =
+            dropdown.inputProperties |> Maybe.isJust
+
+        viewInput inputProps =
+            input
+                [ css
+                    [ outline none
+                    , important (boxShadow none)
+                    , borderStyle none
+                    , fontSize (rem 1)
+                    , height (rem 1.75)
+                    , width (pct 100)
+                    , minWidth (pct 70)
+                    , flexGrow (num 1)
+                    , flexBasis (rem 10)
+                    , paddingLeft (rem 1)
                     ]
+                , placeholder dropdown.placeholder
+                , value currentInput
+                , onInput inputProps.onInput
+                , Attrs.attribute "aria-controls" inputProps.uniqueId
+                , Attrs.attribute "role" "combobox"
+                , Attrs.attribute "aria-expanded"
+                    (if dropdown.hasFocus then
+                        "true"
+
+                     else
+                        "false"
+                    )
                 ]
-                viewOptionGroup
+                []
+
+        icon =
+            case dropdown.inputProperties of
+                Just _ ->
+                    Icon.search
+                        [ css
+                            [ styleIf (Maybe.isJust dropdown.inputProperties) (position absolute)
+                            , left (rem 0.5)
+                            , pointerEvents none
+                            , color Colors.gray
+                            , width (rem 1)
+                            , height (rem 1)
+                            ]
+                        ]
+
+                Nothing ->
+                    Icon.chevronDownFilled []
+
+        selectItems =
+            dropdown.optionGroups |> List.indexedMap (viewSelectItems dropdown) |> List.filterMap identity
     in
     Label.init
         dropdown.label
         Label.GroupLabel
         |> Label.withRequirednessHint dropdown.requirednessHint
         |> Label.withHintText dropdown.hint
         |> Label.view
-            (Events.on "outsideclick" (Decode.succeed (dropdown.onFocus False))
-                :: css [ position relative ]
-                :: attrs
+            ([ Events.on "focusin" (Decode.succeed (dropdown.onFocus True))
+             , css [ position relative ]
+             ]
+                ++ attrs
             )
-            [ OnClickOutsideSupport.view { isActive = dropdown.hasFocus, useNewBehaviour = dropdown.newOutsideClickListener }
+            [ OutsideEventSupport.view { msg = dropdown.onFocus False, isActive = dropdown.hasFocus, eventType = OutsideEventSupport.OutsideClick }
+            , OutsideEventSupport.view { msg = dropdown.onFocus False, isActive = dropdown.hasFocus, eventType = OutsideEventSupport.OutsideFocus }
```

Comment:
Could have the different types on the same element here:

```suggestion
            [ OutsideEventSupport.view { isActive = dropdown.hasFocus, eventTypes = [OutsideEventSupport.OutsideClick (dropdown.onFocus False), OutsideEventSupport.OutsideFocus (dropdown.onFocus False)] }
```

(just a nitpick)